app/scripts/views/complete_sign_up.js

```diff
@@ -70,89 +66,114 @@ define(function (require, exports, module) {
     },
 
     beforeRender () {
-      var verificationInfo = this._verificationInfo;
+      const verificationInfo = this._verificationInfo;
```

Comment:
extract the majority of this cleanup into its own PR to make each PR simpler to review.